client/src/pages/SocketioConnection.js

```diff
@@ -1,20 +1,27 @@
 import React, { useState, useEffect } from 'react';
 import io from 'socket.io-client';
 let endPoint = "http://127.0.0.1:5000/";
-let socket = io.connect(`${endPoint}`);
+let socket = io.connect(null, {port:5000, rememberTransport: false});
```

Comment:
the package `socket.io-client` is not included in the install file so it won't compile

Reply:
It was being merged as part of a previous commit that didn't go through the PR. 

https://github.com/hatchways/team-ninja-turtles/pull/78/files#diff-1846122c2c83a486a3693f7966aa522c34cf489f674185c4da0d9221683fd81fR14

Reply:
It was included as part of the previous merge: